### ISSUE TITLE
Latest SteamVR has broken SpaceCalibrator

### ISSUE THREAD

**Issue by TagPlaysTV** [pushrax/OpenVR-SpaceCalibrator#28]
Any fix for this driver?  Crashes and gets blocked by the latest steamvr.

**Comment by pushrax** [comment]
To clarify, is this a new issue with a SteamVR update in the last couple days, or does the latest Space Calibrator version (v1.1) work for you?

**Comment by TagPlaysTV** [comment]
Yes. 

OpenVR Space Calibrator (Latest version 1.1) is throwing this error for me.

Runtime Error
Space Calibrator Driver unavailable, is SteamVR running and not in safe mode? Error: The System cannot find the file specified.

The .dll is there.  I have uninstalled and re-installed both SteamVR, Open VR Advanced Settings, Space Calibrator and the new DLL for input emulator.

**Comment by pushrax** [comment]
What is your SteamVR version? After you disable safe mode, and unblock the driver, what message does SteamVR say when you launch? Can you post the contents of `space_calibrator_driver.log`? It should be beside the driver DLL.

**Comment by TagPlaysTV** [comment]
SteamVR Hotfix 1.14.16

No error pops when I run: vrmonitor://safemode/enable  

LOG: 

[14:45:40] OpenVR-SpaceCalibratorDriver loaded
[14:45:40] Enabled hook for IVRDriverContext::GetGenericInterface
[14:45:40] Enabled hook for IVRServerDriverHost005::TrackedDevicePoseUpdated
[14:45:40] Enabled hook for IVRServerDriverHost006::TrackedDevicePoseUpdated
[14:45:42] IPC client connected
[14:50:31] IPC client disconnecting normally
[14:50:38] OpenVR-SpaceCalibratorDriver unloaded

So does this mean I can only run SteamVR in safe mode now?

**Comment by pushrax** [comment]
You need to _disable_ safe mode or it won't load the driver.

**Comment by TagPlaysTV** [comment]
Okay, got it.  Added SAFE to the launch options. Which disables safe mode...which confused the heck outta me. Not sure why that was not in there to begin with.  Do you have a cashapp, paypal or venmo?

**Comment by pushrax** [comment]
Hm, that is really strange. I'm not familiar with the SAFE launch option - usually safe mode / driver blocking configuration is done in the SteamVR settings interface. If it's working for you now that's great though.

I'm glad you appreciate the software! I don't have a donation setup right now (if that's what you were getting at), happy to share for free 😄

**Comment by Poolikiki** [comment]
Can I use the previous version of steamVR?

**Comment by pushrax** [comment]
Has there actually been a SteamVR change that broke something? For the OP it appears turning off safe mode fixed it. Safe mode turns on when a driver crashes, and it probably wasn’t Space Calibrator (I would guess the broken input emulator driver before getting the fixed one)

**Comment by Poolikiki** [comment]
I think the latest update to SteamVR broke something, not sure as I'm not the expert.

**Comment by Poolikiki** [comment]
Okay- I've turned off safe mode, and it hasn't fixed the problem, any other solutions? Thanks

**Comment by pushrax** [comment]
Did you already update to Space Calibrator v1.1, released a month ago when the initial breaking change was released in SteamVR beta?

**Comment by TagPlaysTV** [comment]
Hey Poolikiki what I did is called up SteamVR properties.
Click on the GENERAL tab.
The Click on SET LAUNCH OPTIONS...
Add "safe" (without quotes) to the launch options.
Click OKAY
That fixed it to me.

**Comment by Poolikiki** [comment]
okay- thanks

**Comment by Poolikiki** [comment]
Hmmm I tried that, it's still giving me the same error.

**Comment by Poolikiki** [comment]
ah I've fixed it, thanks for helping me!
